ui/src/main/webapp/src/app/project/migration-project-form.component.ts

```diff
@@ -55,6 +56,9 @@ export class MigrationProjectFormComponent extends FormComponent implements OnIn
             }
 
             this.isInWizard = flatRouteData.data.hasOwnProperty('wizard') && flatRouteData.data['wizard'];
+
+            if(!this.editMode) // Creating a new project.
+                this._migrationProjectService.deleteProvisionalProjects().subscribe(res => {});
```

Comment:
I don't remember working on that code. I'll happily listen to what was wrong there when there is time.